config/database.php

```diff
@@ -51,7 +51,7 @@
             'password'       => DB_PASSWORD,
             'unix_socket'    => env('DB_SOCKET', ''),
             'charset'        => DB_CHARSET,
-            'collation'      => DB_COLLATE,
+            'collation'      => $GLOBALS['wpdb']->collate,
             'prefix'         => $table_prefix ?? $GLOBALS['wpdb']->prefix,
```

Comment:
`$table_prefix` will always be `null` here. We could either use `$GLOBAL['table_prefix']` or drop it entirely. I'm leaning toward the latter.

```suggestion
            'prefix'         => $GLOBALS['wpdb']->prefix,
```